rpm/droid-config-xqau51.spec

```diff
@@ -5,6 +5,8 @@
 
 %define pixel_ratio 1.75
 
+Conflicts: appsupport-system-privileged
```

Comment:
Please add this to `droid-config-common.inc`.

Reply:
It's device specific and cannot be in common.

Reply:
Ah you mean the local one, i've put it there now.